articles/130_ros_time.md

```diff
@@ -39,85 +45,110 @@ Slower than real time simulation is necessary for complicated systems where accu
 
-The ability to support pausing time requires that we not assume that the time values are always increasing.
-
-When communicating the changes in time propagation, the latencies in the communication network becomes a challenge.
+When communicating the changes in time propagation, the latencies in the communication network become a challenge.
 Any change in the time abstraction must be communicated to the other nodes in the graph, but will be subject to normal network communication latency.
 This inaccuracy is proportional to the latency of communications and also proportional to the increase in the real time factor.
 If very accurate timestamping is required when using the time abstraction, it can be achieved by slowing down the real time factor such that the communication latency is comparatively small.
+We recommend using synchronized system clock sources for live work on nodes spread across multiple devices.
 
 The final challenge is that the time abstraction must be able to jump backwards in time, a feature that is useful for log file playback.
 This behavior is similar to a system clock after a negative date change, and requires developers using the time abstraction to make sure their algorithm can deal with the discontinuity.
-Appropriate APIs must be provided for to the developer API to enable notifications of jumps in time, both forward and backwards.
+Appropriate APIs must be provided for to the developer API to enable or handle notifications of jumps in time, both forward and backwards.
 
 ### Time Abstractions
 
-There will be at least three versions of these abstractions with the following types, `SystemTime`, `SteadyTime` and `ROSTime`.
-These choices are designed to parallel the [`std::chrono`][] [`system_clock`][] and [`steady_clock`][].
+There will be at least three versions of these abstractions with the following types, `ROSTime`, `SystemTime`, and `SteadyTime`.
+All abstractions provide/utilize the same API.
+The latter two choices are designed to parallel the [`std::chrono`][] [`system_clock`][] and [`steady_clock`][].
 It is expected that the default choice of time will be to use the `ROSTime` source, however the parallel implementations supporting `steady_clock` and `system_clock` will be maintained for use cases where the alternate time source is required.
 
 #### System Time
 
-For convenience in these cases we will also provide the same API as above, but use the name `SystemTime`.
+System Time pulls the current time from the system clock chip. 
+It provides benefit over Steady Time when all nodes in the system have synchronized system time available or when all nodes run on the same host that contains a system clock chip.
 
-`SystemTime` will be directly tied to the system clock.
+We expect a System Time publisher to throw an error if the system clock resource is not available.
+We expect methods returning the current time to fall back to Steady Time if System Time is not available.
```

Comment:
Yes, we definitely want to ensure that our clock code returns the same numbers independently of the operating system.